src/contexts/PromotionsListContext/index.tsx

```diff
@@ -53,7 +53,7 @@ export function PromotionsListProvider({
    */
   useEffect(() => {
     dispatch({ type: DispatchAction.DATA_LOADING });
-    getPromotions(authUser?.user?.id)
+    PromotionService.getPromotions(authUser?.user.id || '')
```

Comment:
```suggestion
    PromotionService.getPromotions(authUser?.user?.id)
```

I don't think we should make it empty string, I would recommend to keep it undefined. Otherwise it'll complain like this:
![image](https://user-images.githubusercontent.com/49849754/114258414-2d999880-997b-11eb-8179-98941ff53f0f.png)

Reply:
Oh yes good catch thank you!!! Will fix